Multitouch Support/VoodooI2CDigitiserTransducer.hpp

```diff
@@ -90,6 +90,7 @@ class EXPORT VoodooI2CDigitiserTransducer : public OSObject {
     
 public:
     DigitiserTransducerButtonState physical_button;
+    bool has_secondary_button = false;
```

Comment:
I'll create a thread here because I can't do it on the satellite diff:
We can't just change the VoodooI2CHID commit, it's not part of its `master` branch :P

BTW, do we really need to set `has_secondary_button` on every report? It will never change.